src/PostgREST/Config.hs

```diff
@@ -146,50 +146,56 @@ readPathShowHelp = customExecParser parserPrefs opts
 
     exampleCfg :: Doc
     exampleCfg = vsep . map (text . toS) . lines $
-      [str|db-uri = "postgres://user:pass@localhost:5432/dbname"
-          |db-schema = "public" # this schema gets added to the search_path of every request
+      [str|REQUIRED:
```

Comment:
How about adding this as a comment:
```suggestion
      [str|##REQUIRED:
```

#### Reasoning

I'd like to create a sample config in a quick step with `postgrest 2> postgrest.conf`. Right now it's not entirely usable, but it's something.

I also added that trick in http://postgrest.org/en/v7.0.0/install.html#running.

Reply:
Absolutely, I had that in mind as well, but I missed it when adding those to. I'll comment them out. Maybe even with `###` because it's one more level then the regular comments.